webpack.config.js

```diff
@@ -5,8 +5,19 @@ const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const HtmlWebPackPlugin = require('html-webpack-plugin');
 
 module.exports = {
+    entry: ['babel-polyfill', './src/index.js'],
     devServer: {
         contentBase: './dist',
+        open: true,
+        historyApiFallback: {
+            index: 'index.html',
+        },
+        headers: {
```

Comment:
Why do you need to support cors here?
Was that implemented just for oauth purposes?

Reply:
This part of the config is useless, and I will remove it.

Reply:
Done.